### ISSUE TITLE
Doesn't support 2FA?

### ISSUE THREAD

**Issue by simonsolberg** [jenius-apps/beeskie#9]
I tried to login but the app throws an error message below the login info. I receive an e-mail with my 2FA code from bluesky, but there is no place in the application to input the 2FA code.

**Comment by dpaulino** [comment]
2FA isn't supported yet! But I will add it soon. Thank you for making a report so I can track it

**Comment by dpaulino** [comment]
@simonsolberg according to bluesky API documentation, I don't think that the 2FA code is required if you sign in using an app password (not your real account password). So I want to double check, did you try to log in to Beeskie with an app password? 

More details on app passwords here: https://github.com/jenius-apps/beeskie/blob/main/docs/appPasswordsHelp.md

**Comment by simonsolberg** [comment]
You are correct that works. Maybe the app should guide the user to creating an app password when 2FA is enabled on the account?
